server/settings/publish_plugins.py

```diff
@@ -1428,6 +1446,9 @@ class PublishPuginsModel(BaseSettingsModel):
     "AttachReviewables": {
         "enabled": True,
     },
+    "CollectRenderedFiles": {
+        "remove_files": True
```

Comment:
If I understood correctly from @philippe-ynput we actually, moving forward, want to disable this removing behavior by default. If so, this suggestion should be applied.

```suggestion
        "remove_files": False
```